app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -341,10 +341,6 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         settings().shouldReturnToBrowser =
             components.core.store.state.getNormalOrPrivateTabs(private = false).isNotEmpty()
 
-        if (settings().lastKnownMode.isPrivate) {
```

Comment:
I think we still need this because of 
https://github.com/mozilla-mobile/fenix/issues/11153
Even if screenshots are allowed
We should do our best to not display the private content in the recents screen.

Reply:
If we keep this a counterpart in onResume will probably be needed.

Reply:
Oops totally forgot about this behavior. Good call

Reply:
Added comments explaining this too to avoid regressions.